src/StripeTests/Services/GeneratedExamplesTest.cs

```diff
@@ -1339,6 +1339,38 @@ public void TestPaymentIntentServiceUpdate()
             service.Update("pi_xxxxxxxxxxxxx", options);
         }
 
+        [Fact]
+        public void TestPaymentLinkServiceCreate()
+        {
+            var options = new PaymentLinkCreateOptions
+            {
+                LineItems = new List<PaymentLinkLineItemOptions>
+                {
+                    new PaymentLinkLineItemOptions
+                    {
+                        Price = "price_xxxxxxxxxxxxx",
+                        Quantity = 1,
+                    },
+                },
+            };
+            var service = new PaymentLinkService(this.StripeClient);
+            service.Create(options);
+        }
+
+        [Fact]
+        public void TestPaymentLinkServiceListLineItems()
+        {
+            var service = new PaymentLinkService(this.StripeClient);
+            service.ListLineItems("pl_xyz");
```

Comment:
Those tests are quite lacking. Old tests checked the return class and its object property and all that no?

Reply:
I agree. Checking the return class seems a little redundant since that is already enforced by the static type, but I'll look into seeing if we can add object property checks to these.